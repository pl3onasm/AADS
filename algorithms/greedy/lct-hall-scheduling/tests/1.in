[5,9), [1,4), [10,12), [5,8), [11,15), [3,6), [4,6), [10,15), [0,3)