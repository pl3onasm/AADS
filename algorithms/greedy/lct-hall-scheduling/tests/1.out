Number of halls: 4

SCHEDULES

---------
 Hall 1
---------
Act 9: [ 0, 3 )
Act 6: [ 3, 6 )
Act 8: [ 10, 15 )

---------
 Hall 2
---------
Act 2: [ 1, 4 )
Act 7: [ 4, 6 )
Act 3: [ 10, 12 )

---------
 Hall 3
---------
Act 1: [ 5, 9 )

---------
 Hall 4
---------
Act 4: [ 5, 8 )
Act 5: [ 11, 15 )

